docs-ref-services/machine-learning.md

```diff
@@ -18,16 +18,36 @@ ms.service: Machine Learning
 ## Overview
 Machine learning is a technique of data science that helps computers learn from existing data in order to forecast future behaviors, outcomes, and trends. These forecasts or predictions from machine learning can make apps and devices smarter. When you shop online, machine learning helps recommend other products you might like based on what you've purchased. When your credit card is swiped, machine learning compares the transaction to a database of transactions and helps detect fraud. When your robot vacuum cleaner vacuums a room, machine learning helps it decide whether the job is done.
 
-## Install the modules with npm
+## Management
 
 Use npm to install the Azure Machine Learning modules for Node.js
 
-### Management
+### Install the modules with npm
 ```bash
 npm install azure-arm-machinelearning
 ```
-##Example
+### Example
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const MachineLearningManagement = require('azure-arm-machinelearning');
 
-##Samples
+msRestAzure
+  .interactiveLogin()
+  .then(credentials => {
+    const subscriptionId = 'your-subscription-id';
+    const client = new MachineLearningManagement.CommitmentPlansManagementClient(
+      credentials,
+      subscriptionId
+    );
+    return client.commitmentPlans.list();

```

Comment:
is this a promise? if so, this is fine. if not, then i woudl combine this code with the following `then`